services/src/test/scala/cromwell/services/instrumentation/impl/stackdriver/TestStackdriverInstrumentationServiceActor.scala

```diff
@@ -0,0 +1,30 @@
+package cromwell.services.instrumentation.impl.stackdriver
+
+import akka.actor.ActorRef
+import com.google.monitoring.v3.CreateTimeSeriesRequest
+import com.typesafe.config.Config
+import scala.collection.JavaConverters._
+
+class TestStackdriverInstrumentationServiceActor(serviceConfig: Config, globalConfig: Config, serviceRegistryActor: ActorRef)
+  extends StackdriverInstrumentationServiceActor(serviceConfig, globalConfig, serviceRegistryActor) {
+
+  var metricsReceived = List[TimeSeriesRequest]()
+
+  override def sendTimeSeriesToStackdriver(timeSeriesRequest: CreateTimeSeriesRequest) = {
+    val timeSeries = timeSeriesRequest.getTimeSeries(0)
+    val metric = timeSeries.getMetric
+
+    val abc = TimeSeriesRequest(metric.getType,
```

Comment:
`abc`?

Reply:
Ugh. I forgot to clean this up. Will do it now.

Reply:
Ugh. I forgot to clean this up. Will do it now.